api/pom.xml

```diff
@@ -136,9 +126,38 @@
 
     <profiles>
         <profile>
-            <id>jdk9</id>
+          <id>jdk9</id>
+          <activation>
+              <jdk>9</jdk>
```

Comment:
I'd just remove this profile and focus on 8 and 11 (LTS versions) only. If you think this should not be removed then profile for JDK 10 should be added..

Reply:
+1 for only doing Java 8 and 11

Reply:
Fine, I'll fix it. I added java9 profile because we used jdk9 to build the project before. Removing it will make pom.xml simpler and more readable.